docs/internal-documentation/distrib-testing/Centos-preparation.md

```diff
@@ -0,0 +1,43 @@
+# CentOS system preparation for installing Zonemaster components
+
+1. Make a clean installation of CentOs.
+
+2. Update the package database.
+
+   ```sh
+   yum update
+   yum groupinstall "Development Tools"
+   ```
+
+3. Install prerequisites
+
+   ```sh
+   yum install cpanminus
+   ```
+
+4. Create a new user.
+
+   ```sh
+   adduser username
+   passwd username
+   ```
+
+5. Add user to the sudoer list.
+
+   ```sh
+   usermod -aG wheel username
+   ```
+
+6. Exit from the root shell.
+
+   ```sh
+   exit
+   ```
+
+
+## Preparing to install the repositories
+
+1. Follow the steps (1-7) of the [release
+process](https://github.com/dotse/zonemaster/blob/master/docs/internal-documentation/maintenance/ReleaseProcess.md) to prepare the tarballs in a separate machine 
```

Comment:
I think we want to link to the ReleaseProcess document in the same branch as the document as one is reading:

```
1. Follow the steps (1-7) of the [release process] to prepare the tarballs in a separate machine

[release process]: ../maintenance/ReleaseProcess.md
```

Reply:
@mattias-p, I do not agree with you. The preparations are to create a machine to test on. The ReleaseProcesses is to create packages to install. I think the document is fine if the last link could be updated to point at the ReleaseProcess document in the same branch.

Reply:
@matsduf Testing is an important part of the release process, not the other way around.